From 5c90c94cb0215c496bef63315869a8f1e8c0bcda Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Thu, 17 Nov 2011 12:51:27 +0800
Subject: ENGR00162335-2 [mx6q_arm2]SD card 3.0 can't be recognized

change mmc rescan delay to 500ms from 200ms.
this will make insert and removal reflection more stable.

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 drivers/mmc/host/sdhci.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci.c b/drivers/mmc/host/sdhci.c
index 7ff336b..1f130db 100755
--- a/drivers/mmc/host/sdhci.c
+++ b/drivers/mmc/host/sdhci.c
@@ -1906,7 +1906,7 @@ static void sdhci_tasklet_card(unsigned long param)
 out:
 	spin_unlock_irqrestore(&host->lock, flags);
 
-	mmc_detect_change(host->mmc, msecs_to_jiffies(200));
+	mmc_detect_change(host->mmc, msecs_to_jiffies(500));
 }
 
 static void sdhci_tasklet_finish(unsigned long param)
-- 
1.7.9.5

